### PR TITLE
Bug #164674 fix: search box for cert-xxxx is showing error

### PR DESCRIPTION
**Pull request by tushar33**

### CODE REVIEW

File: src/components/com_tjcertificate/site/views/certificate/view.html.php [hunk 1]
```diff
@@ -70,13 +70,13 @@ public function display($tpl = null)
...
-			if (!$this->certificate->id)
-			{
-				JError::raiseWarning(500, Text::_('COM_TJCERTIFICATE_ERROR_CERTIFICATE_EXPIRED'));
+		if (!$this->certificate->id)
+		{
+			JError::raiseWarning(500, Text::_('COM_TJCERTIFICATE_ERROR_CERTIFICATE_EXPIRED'));
 
-				return false;
-			}
+			return false;
```
Comment (praneettekdi) [inline src/components/com_tjcertificate/site/views/certificate/view.html.php:79]: Don't return false, it won't show me the search box again. Also, did you check if the image generation code is running if the certificate is not present @tushar33

Comment (tushar33) [reply src/components/com_tjcertificate/site/views/certificate/view.html.php]: done

File: src/components/com_tjcertificate/site/views/certificate/view.html.php [hunk 2]
```diff
@@ -72,47 +72,48 @@ public function display($tpl = null)
...
 			$this->certificate = $certificate::validateCertificate($this->uniqueCertificateId);
+		}
+
+		if (!$this->certificate->id)
+		{
+			JError::raiseWarning(500, Text::_('COM_TJCERTIFICATE_ERROR_CERTIFICATE_EXPIRED'));
+		}
 
-			if (!$this->certificate->id)
+		if ($this->certificate->id)
```
Comment (praneettekdi) [inline src/components/com_tjcertificate/site/views/certificate/view.html.php:82]: use elseif

Comment (tushar33) [reply src/components/com_tjcertificate/site/views/certificate/view.html.php]: done

**Pull request merged**